packages/walletconnect-v1/src/wc-wallet.ts

```diff
@@ -102,6 +102,7 @@ export class WalletConnectWallet extends RemoteWallet<WalletConnectSigner> {
     return this.client?.uri;
   }
 
+  // These functions appear to be overwritten elsewhere (at least I dont anything happening in the logs)
```

Comment:
That's correct, that's the way the wc-v2 was built and this is carbon copy of it. I also think we should just refactor this to register event listeners rather than overwrite the default listener